README.md

```diff
@@ -1,7 +1,11 @@
 ## MPL Docker image for building docs
 
+[![GitHub Actions Status: CI](https://github.com/matplotlib/mpl-docker/workflows/CI/badge.svg)](https://github.com/matplotlib/mpl-docker/actions?query=workflow%3ACI+branch%3Amaster)
+[![Docker Pulls](https://img.shields.io/docker/pulls/matplotlib/mpl-docker)](https://hub.docker.com/r/matplotlib/mpl-docker)
+[![Docker Image Size (tag)](https://img.shields.io/docker/image-size/matplotlib/mpl-docker/latest)](https://hub.docker.com/r/matplotlib/mpl-docker/tags?name=latest)
```

Comment:
@QuLogic These obviously don't render yet, as there isn't anything there until this PR gets merged to `master`. However, these patterns do work. I can show examples for some of my other projects if that is useful.